docs/products/cassandra/howto/connect-go.rst

```diff
@@ -0,0 +1,45 @@
+Connect with Go
+---------------
+
+This example connects to an Aiven for Apache Cassandra® service from Go as the ``avnadmin`` user by making use of the ``gocql`` library. 
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
```

Comment:
Is this accurate? these are command line flags rather than placeholders in the code.. .

Reply:
I believe this is the terminology we used also in other pages, I would state is ok :)